setup.py

```diff
@@ -6,12 +6,12 @@
     author='Feature Labs, Inc.',
     author_email='support@featurelabs.com',
     license='MIT',
-    install_requires=['tsfresh>=0.11.2', 'featuretools>=0.7.0'],
+    install_requires=['tsfresh>=0.11.2', 'featuretools>=0.11.0'],
```

Comment:
I think we can keep the featuretools minimum requirement at 0.7.0 since all versions since then are compatible with this library

Reply:
changed version back to 0.7.0